chainstate_test_framework/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "chainstate_test_framework"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+chainstate = { path = '../chainstate' }
+chainstate-storage = { path = '../chainstate-storage' }
+chainstate-types = { path = '../chainstate-types' }
+common = { path = '../common' }
+crypto = { path = '../crypto' }
+rstest = "0.15"
```

Comment:
We're using the arbitrary convention to separate external crates with a new line from our internal crates